src/bh_route_pending_txns.erl

```diff
@@ -44,23 +44,31 @@ handle('POST', [], Req) ->
 handle(_, _, _Req) ->
     ?RESPONSE_404.
 
--spec insert_pending_txn(#blockchain_txn_payment_v1_pb{}, binary()) -> {ok, jsone:json_object()}.
-insert_pending_txn(#blockchain_txn_payment_v1_pb{
-                      payer=Payer,
-                      nonce=Nonce
-                     }=Txn, Bin) ->
+-type supported_txn() :: #blockchain_txn_payment_v1_pb{}
+                         | #blockchain_txn_payment_v2_pb{}.
+-type nonce_type() :: binary().
```

Comment:
why is the nonce_type binary?

Reply:
because erlang types don't like something like:

```
-type nonce_type() :: <<"balance">>, <<"security">>, <<"dc">>.
```
 The binary string is what gets submitted to the database (enum nonce_type)

Reply:
huh, okay. probably fine then.